src/content/docs/new-relic-solutions/new-relic-one/ui-data/service-maps/service-maps.mdx

```diff
@@ -0,0 +1,165 @@
+  src={SolutionsServiceMaps}
+/>
+
+<figcaption>
+  **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > APM > (choose an entity) > Service map**: Service map shows your dependencies and how they're performing.
+</figcaption>
+
+## Use service map [#using-service-maps]
+
+<Callout variant="tip">
+  Service map works with [distributed tracing](/docs/apm/distributed-tracing/getting-started/introduction-distributed-tracing) to connect relationships between entities. Service map is still functional if you haven't enabled distributed tracing, but we recommend having distributed tracing enabled for all agents. This ensures a more consistent experience while using service maps.
+  </Callout>
+
+To access service map: 
+1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities)**, then select either **APM & services**, **Mobile**, or **Browser**.
+2. Select an entity.
+3. On the left-hand side, click **Service map**.
+
+### Map from one entity [#map-from-one-entity]
+
+Hover your cursor over an entity and click "Map from this entity" to view how entities are related from that entity's perspective. The service map will update accordingly.
+
+<img
+  title="Map from this entity"
+  alt="A screenshot of the tooltip that appears when you hover your cursor over an entity."
+  src={solutionsMapEntity}
+  style={{ width: "70%" }}
+/>
+
+<img
+    title="Example of map from this entity"
+    alt="A screenshot of how service map updates after mapping from a specific entity."
+    src={solutionsMapfromthisentityexample}
+/>
+
+### Filter entities [#filter-map]
+
+You can filter the service map based:
+
+* **Health status**: Remove the health status color around affected entities.
+* **Entity types**: Hide specific entities from the map.
+* **Relationship depth**:
+  * **Depth 1**: (Default) Displays all entities directly connected to the main entity, regardless of their health status.
+  * **Depth 2**: Focuses on degraded entities (with alerts) that are two hops away from the main entity. Entities with direct connections to the main entity are not included in this view.
+  * **Depth 3**: Similar to depth 2, depth 3 concentrates on degraded entities with alerts but extends the search to entities that are three hops away from the main entity. Direct connections to the main entity remain excluded.
+
+  <Callout variant="tip">
+  The purpose of limiting the display of entities to degraded ones in depths 2 and 3 is to prevent the service map from becoming cluttered and overwhelming. As you increase the relationship depth, you are going further into the network of entities, but you are only seeing entities that are experiencing issues. This can help you to focus on the most critical problems in your system.
+    </Callout>
+ * **Timewarp**: Explore how entity health changed in the last three hours. 
```

Comment:
Move this timewarp section above the depth sections so the callout ends the list